packages/core/src/prompts/password.ts

```diff
@@ -10,7 +10,7 @@ export default class PasswordPrompt extends Prompt {
 	get cursor() {
 		return this._cursor;
 	}
-	private get masked() {
+	get masked() {
```

Comment:
Unrelated to this PR, but just curious: any particular reason we don't use `replaceAll()` here?

`return this.value.replaceAll(/./g, this._mask);`

Easier to parse/read IMO.

Reply:
Yes, sorry. They were intended for him. Should have @mentioned him.

Reply:
No particular reason! I'm cool with that change.